modular_zubbers/code/modules/antagonists/modglue.dm

```diff
@@ -0,0 +1,9 @@
+/obj/item/mod/control/pre_equipped/contractor/set_wearer(mob/living/carbon/human/user)
+	. = ..()
+	var/obj/item/I = src
+	ADD_TRAIT(I, TRAIT_NODROP, GLUED_ITEM_TRAIT)
```

Comment:
I don't think ``I`` needs to be casted here, and ``I`` can just be ``src`` in the ADD_TRAIT define